pkg/asset/machines/master.go

```diff
@@ -292,6 +294,12 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 				}
 			}
 		}
+	case ovirttypes.Name:
+		pool.Platform.Ovirt = &ovirttypes.MachinePool{}
```

Comment:
This should support the default platform convention like the other platforms.

Reply:
done